dev/benchmarks/complex_layout/android/app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@ found in the LICENSE file. -->
 
     <uses-permission android:name="android.permission.INTERNET"/>
 
-    <application android:name="${applicationName}" android:label="complex_layout" android:icon="@mipmap/ic_launcher">
+    <application android:name="io.flutter.app.FlutterApplication" android:label="complex_layout" android:icon="@mipmap/ic_launcher">
         <activity android:name="io.flutter.app.FlutterActivity"
```

Comment:
The issue seems to be that this needs to be changed to `io.flutter.embedding.android.FlutterActivity`. fyi @GaryQian